### PR TITLE
Update class-01 README

### PR DESCRIPTION
**Pull request by patci-zz**
To increase clarity and provide updated instructions.

### CODE REVIEW

**Comment by brookr** [comment]
Thanks for this!

Why is the class-16 readme getting deleted in this PR?

File: class-01-jquery-and-the-dom/videos.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Class 1 - 301d2
```
Comment (brookr) [inline class-01-jquery-and-the-dom/videos.md:1]: This upstream template doesn't need video links.

**Comment by patci-zz** [comment]
Strange. Fixing this on my end.

**Comment by brookr** [comment]
Pull the latest, got the `cp -r` commands in. Thanks for catching that! You can PR the git url instructions still if you like.

**Pull request closed without merging**